sources/packages/backend/apps/queue-consumers/src/queue-consumers.module.ts

```diff
@@ -35,6 +40,12 @@ import { TypeOrmModule } from "@nestjs/typeorm";
     StudentAssessmentService,
     SshService,
     QueueService,
+    CRAResponseIntegrationScheduler,
+    CRAProcessIntegrationScheduler,
+    SequenceControlService,
+    CRAIncomeVerificationsService,
```

Comment:
The `CRAIncomeVerificationsService` is not supposed to be exported from `CRAIntegrationModule`?